public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.ts

```diff
@@ -42,6 +44,7 @@ export interface LoadDashboardOptions {
   uid: string;
   route: DashboardRoutes;
   urlFolderUid?: string;
+  queryParams?: BackendSrvRequest['params'];
```

Comment:
I see ` BackendSrvRequest['params'];` in a few places, this to me is pretty unclear. I would use  `Record<string, any>` or UrlQueryMap

Reply:
Totally forgot about `UrlQueryMap`. Thanks! I just pushed an update with the changes.